### PR TITLE
Vacuum blog post

### PR DESCRIPTION
**Pull request by nkarpov**
Redoing this one after the accidental overwrite

### CODE REVIEW

File: src/blog/2023-01-03-delta-lake-vacuum-command/index.mdx [hunk 1]
```diff
@@ -0,0 +1,240 @@
+---
+title: Delta Lake Vacuum Command
+description: This blog post explains how to vacuum files marked for deletion from storage with the Delta Lake Vacuum command.
+thumbnail: ./thumbnail.png
+author: Matthew Powers, Nick Karpov
+---
+
+# Delta Lake Vacuum Command
```
Comment (dennyglee) [inline src/blog/2023-01-03-delta-lake-vacuum-command/index.mdx:8]: Could you remove this as this is the title of the post and we already have this from the `title` command above

**Review by dennyglee — CHANGES_REQUESTED** [review]

File: src/blog/2023-01-03-delta-lake-vacuum-command/index.mdx (hunk 1 above)
Comment (nkarpov) [reply src/blog/2023-01-03-delta-lake-vacuum-command/index.mdx]: Done, but FYI this header is in a bunch of the other blogs (https://delta.io/blog/2022-12-22-reading-delta-lake-tables-polars-dataframe/ for ex.)

Comment (dennyglee) [reply src/blog/2023-01-03-delta-lake-vacuum-command/index.mdx]: thanks - and we should probably create a GitHub issue for this ;)

**Comment by dennyglee** [comment]
thanks @nkarpov - could you check as it's stating there is a branch conflict.  thanks!

**Pull request closed without merging**